### PR TITLE
[stable/datadog]

### PR DESCRIPTION
**Pull request by denniswebb**
Signed-off-by: Dennis Webb <dennis@bluesentryit.com>

#### What this PR does / why we need it:
Need the ability to add annotations to the cluster-agent pods, specifically for disabling Istio sidecar injection.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/datadog/values.yaml [hunk 1]
```diff
@@ -360,6 +360,12 @@ clusterAgent:
   #   exec:
   #     command: ["/bin/true"]
 
+  ## @param podAnnotations - list of key:value strings - optional
+  ## Annotations to add to the cluster-agents's pod(s)
+  #
+  # podAnnotations:
+  #   <POD_ANNOTATION>: '[{"key": "<KEY>", "value": "<VALUE>"}]'
```
Comment (davidkarlsen) [inline stable/datadog/values.yaml]:
```suggestion
  #   key: "value"
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Review by clamoriniere — APPROVED** [review]
/lgtm

**Pull request merged**